### PR TITLE
Initialize eslint

### PR DESCRIPTION
**Pull request by a--hoang**
References #16 .  This is a very simple config file for eslint that tries to follow current conventions on gdax-node.  It produces ~1000 linting errors that I can also include in the PR, if you like.

Btw, hi from one of your future interns!

### CODE REVIEW

**Comment by a--hoang** [comment]
You can now lint with "npm run lint" and add more rules to the .eslint.json file.

**Comment by a--hoang** [comment]
You can now lint with "npm run lint" and add more rules to the .eslint.json file.

File: lib/clients/authenticated.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 var util = require('util');
-var querystring = require('querystring');
```
Comment (a--hoang) [inline lib/clients/authenticated.js]: Unused import

File: lib/clients/authenticated.js [hunk 2]
```diff
@@ -206,9 +205,9 @@ _.assign(AuthenticatedClient.prototype, new function() {
         }
 
         if ((resp && resp.statusCode != 200) || !data) {
-          var err = new Error('Failed to cancel all orders');
+          var error = new Error('Failed to cancel all orders');
```
Comment (a--hoang) [inline lib/clients/authenticated.js]: Redeclared err from above (line 202)

**Comment by awitherow** [comment]
Give up @a--hoang ?

**Comment by a--hoang** [comment]
@awitherow I've been told that they are developing their own internal eslint config and are trying to standardize all their Node stuff with it...making this unfit for PR :( I will, however, PR some of the stuff that was found by the linter (redeclared variables, etc)

**Pull request closed without merging**